src_lockfree/michael_scott_queue.ml

```diff
@@ -35,30 +35,30 @@ exception Empty
 
 let pop_opt { head; _ } =
   let b = Backoff.create () in
```

Comment:
Hmm... Probably won't make a difference (assuming the compiler is able to constant fold the `Backoff.create ()` call), but it might be a bit safer / more future proof to use the constant `Backoff.default`.  (The same for all the other uses of `Backoff.create ()`.)